### PR TITLE
Add label with num of shown elements in the table

### PR DESCRIPTION
**Pull request by gnaponie**
I would like to point out that I don't really like the "margin-left" part. But I couldn't find a nicer way to do that...
Did you ever consider using "flex-box" to align elements?

### CODE REVIEW

**Comment by mprahl** [comment]
@gnaponie flex-box would be good, but I think you'd have to have a flex container after the "Show Columns" button with all the text elements in it.

**Comment by hanzz** [comment]
I have no idea how to replace the margin-left, but I've checked the UI with this applied and it looks OK in my Firefox browser, so +1 for the change.

**Comment by gnaponie** [comment]
Can we merge this one as it is and then I'll create another PR for flex-box?

File: src/app/tables/table.component.css [hunk 1]
```diff
@@ -138,6 +138,12 @@
     display: none;
 }
 
+.estuary-table-header__num-elements {
+    margin-left: 36rem;
```
Comment (mprahl) [inline src/app/tables/table.component.css]: Why is this margin set? In both Chrome and Firefox it causes the text to be way to the right of page and off the table. Without this margin, the text looks fine.

Comment (gnaponie) [reply src/app/tables/table.component.css]: It looks fine for me (it appears at the end of the table -on the right-, inside the table. I liked that was on the right.
But I don't like it much (the fact that it is a fixed margin). I can just remove it and when I'll try to use flex box we can see if it looks better on the right.

**Comment by gnaponie** [comment]
Rebased.

**Comment by hanzz** [comment]
I think you plan another PR to align it to the right, right? :) I don't like how it looks in UI now, but technically the PR is still OK to me :)

**Comment by gnaponie** [comment]
> I think you plan another PR to align it to the right, right? :) I don't like how it looks in UI now, but technically the PR is still OK to me :)

Eheh, yeah. I can work on the other PR today.

**Comment by gnaponie** [comment]
I've added a commit that refactor the table header with flex box.
And now the thingy with the number of elements it's at right, so also Jan likes it (me too actually) :)

File: src/app/tables/table.component.css [hunk 2]
```diff
@@ -58,20 +57,14 @@
         margin-bottom: 3rem;
     }
 
-    .estuary-table-header {
-        /* Make the width 100% minus the padding used by div.estuary-container */
-        width: calc(100% - 10rem);
+    .estuary-inner-container {
+        max-width: 350px;
+        min-width: 10px;
```
Comment (mprahl) [inline src/app/tables/table.component.css]: Can you set the min-width to 640rem or pixels. That will keep things from getting squished if you only have one small column activated.

**Comment by gnaponie** [comment]
It should be ok now.

File: src/app/tables/table.component.html [hunk 3]
```diff
@@ -7,81 +7,85 @@ <h2 class="title">
...
-        <!-- Create a dropdown menu with all the columns the user can choose to have shown -->
-        <ul *dropdownMenu class="dropdown-menu estuary-table-header__dropdown-menu">
-          <li *ngFor="let column of columns | keyvalue">
-            <!-- The default columns will be checked by default, any changes made by the user will trigger the
-                 setColumnState method to be called, which will store that state change in the columns component property. 
-                 This change will then be reflected by Angular when it reruns the various ngFors in this template. -->
-            <input type="checkbox" [checked]="column.value" (change)="setColumnState(column.key)" /> {{ column.key }}
-          </li>
-        </ul>
+  <div class="estuary-inner-container">
```
Comment (mprahl) [inline src/app/tables/table.component.html]: Could you call this `estuary-table-container`?

**Comment by mprahl** [comment]
LGTM

**Pull request merged**